internal/js-dependency-scanner/src/main/java/org/wso2/security/tools/scanner/dependency/js/postprocessor/VulnerabilityReportManager.java

```diff
@@ -0,0 +1,96 @@
+ *   Version 2.0 (the "License"); you may not use this file except
+ *   in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ * /
+ */
+
+package org.wso2.security.tools.scanner.dependency.js.postprocessor;
+
+import org.apache.log4j.Logger;
+import org.eclipse.jgit.api.errors.GitAPIException;
+import org.wso2.security.tools.scanner.dependency.js.constants.JSScannerConstants;
+import org.wso2.security.tools.scanner.dependency.js.exception.ConfigParserException;
+import org.wso2.security.tools.scanner.dependency.js.exception.FileHandlerException;
+import org.wso2.security.tools.scanner.dependency.js.exception.IssueCreatorException;
+import org.wso2.security.tools.scanner.dependency.js.issuecreator.IssueCreator;
+import org.wso2.security.tools.scanner.dependency.js.reportpublisher.ReportUploader;
+import org.wso2.security.tools.scanner.dependency.js.utils.ConfigParser;
+
+import java.util.Arrays;
+import java.util.HashMap;
+
+
+/**
+ * Class to handle publishing scan report in multiple ways.
+ */
+public class VulnerabilityReportManager {
+    private static final Logger log = Logger.getLogger(VulnerabilityReportManager.class);
+
+    public VulnerabilityReportManager() {
+    }
+
+    /**
+     * This method handle all ways to publish the output.
+     *
+     * @param productResponseMapper mapper for product and scan result.
+     */
+    public void controlReportManager(HashMap<String, String> productResponseMapper) {
+        //Instance to upload scan report to github repo
+        log.info("[JS_SEC_DAILY_SCAN]  " + "Start uploading report to git repo.");
+        ReportUploader reportUploader = null;
+        try {
+            reportUploader = ConfigParser.parseGitUploaderConfigProperties();
+        } catch (GitAPIException e) {
+            log.error("Upload scan reports to git repository failed. " +
+                    "Unable to clone or pull Security Artifact repository", e);
+        } catch (ConfigParserException e) {
+            log.error("Upload scan reports to git repository failed.", e);
+        }
+        try {
+            if (reportUploader != null) {
+                reportUploader.publishReport(productResponseMapper);
+            }
+        } catch (GitAPIException | FileHandlerException e) {
+            log.error("Upload scan reports to git repository failed.", e);
+        }
+
+        log.info("[JS_SEC_DAILY_SCAN]  " + "Successfully upoloaded reports to git repo.");
+        //Instance to create jira ticket
+        log.info("[JS_SEC_DAILY_SCAN]  " + "Start creating JIRA tickets.");
```

Comment:
"//Instance to create jira ticket" comment should come after this info log, immediately before the variable declaration. Also, this comment is actually not necessary, because the usage of the variable is obvious from the variable name. Having said that, as mentioned previously, use either "ticket" or "issue", don't use them interchangeably in the code. It is confusing.